omnigibson/simulator.py

```diff
@@ -748,13 +747,30 @@ def adding_objects(self, objs):
             Args:
                 objs (Iterable[BaseObject]): list of objects to add
             """
-            # Immediately yield
+            SimulationManager = lazy.isaacsim.core.simulation_manager.SimulationManager
```

Comment:
It makes no sense to have this context with the objs arg, because if you already know how many objects you're importing you can just call post_import yourself. Let's leave a TODO here to remove this context and call post_import directly.